javascript/ql/test/library-tests/Portals/src/typeTracking/prototypeFlow.js

```diff
@@ -0,0 +1,48 @@
+// example adapted from test cases found in github projects
```

Comment:
For examples based on third-party code, we generally provide a reference for where we found it and what the license of the code is. That seems like overkill in this case. Could you perhaps replace this with a synthetic test that tests the same functionality, but is _not_ derived from other code?

Reply:
Good point, thanks! New commit makes the examples smaller and no longer taken from other github projects.